recipes/oarfish/meta.yaml

```diff
@@ -0,0 +1,34 @@
+{% set version = "0.2.0" %}
+
+package:
+  name: oarfish
+  version: {{ version }}
+
+build:
+  number: 0
+  run_exports:
+    - {{ pin_subpackage("oarfish", max_pin="x.x") }}
+    
+source:
+  url: https://github.com/COMBINE-lab/oarfish/archive/v{{ version }}.tar.gz
+  sha256: 47e9ad6ae8af5f96eac16f294052319948a25892a9554c36ac5fe73646f20c2e
+
+requirements:
+  build:
+    - rust >=1.75  
```

Comment:
can you please try to use `compiler('rust')`

Reply:
Sure!